app.js

```diff
@@ -23,21 +23,19 @@ app.use(express.static(path.join(__dirname, 'public')));
 
 app.use('/', index);
 // catch 404 and forward to error handler
-app.use(function(req, res, next) {
-  var err = new Error('Not Found');
+app.use((req, res, next) => {
+  const err = new Error('Not Found');
   err.status = 404;
   next(err);
 });
 
-// error handler
-app.use(function(err, req, res, next) {
-  // set locals, only providing error in development
+app.use((err, req, res, next) => {
   res.locals.message = err.message;
   res.locals.error = req.app.get('env') === 'development' ? err : {};
```

Comment:
It will break because you can't use req and app object together like this `req.app.get('env')`.  instead of it, use `app.get('env')`